### PR TITLE
 unnecessary console.logs removed

### PR DESCRIPTION
**Pull request by ishanisri**

### CODE REVIEW

File: public/src/index.ts [hunk 1]
```diff
@@ -41,9 +40,7 @@ async function handleStockPricesStream() {
 
     for await (const stockPricesUpdate of stream) {
         const update = stockPricesUpdate.getPricesMap();
-        console.log("Stock prices update");
         update.forEach((newPrice, stockId) => {
```
Comment (coderick14) [inline public/src/index.ts]: Remove the `forEach` loop as well, since the only statement in it has been removed.

**Comment by gauthamk97** [comment]
You need to amend your existing commit, not add a 2nd commit. This is a change that should be done in 1 commit, not 2. (We want to keep the commit history as clean as possible)

To fix this, you'll have to run the following :-
1. Go back in time to before you committed your changes : 
`git reset --soft origin/master` 
The _soft_ flag ensures your changes will still remain, but your commits will be gone. This allows you to rewrite these commits.

2. Stash your changes (Google if you don't know what git stash is)
`git stash`
You are doing this because you're about to perform step 3

3. Pull the current version of the repo (This is a good practice to do before you send any pull request. It especially helps in dealing with potential merge conflicts)
`git pull --rebase origin master`

4. Pop your stashed changes
`git stash pop`

5. Add all the files you've changed
`git add <all_files_you've_changed>`

6. `git commit -m '<insert commit message>`

7. `git push origin frontend-issues`

The pull request will update automatically.

**Note** : Run `git status` in between each step to understand what you're really doing.

File: public/src/components/mortgage/Mortgage.tsx [hunk 2]
```diff
@@ -112,7 +112,6 @@ export class Mortgage extends React.Component<MortgageProps, MortgageState> {
             mortgageStocksRequest.setStockQuantity(stockQuantity);
 
             const resp = await DalalActionService.mortgageStocks(mortgageStocksRequest, this.props.sessionMd);
-            console.log(resp.getStatusCode(), resp.toObject());
             // notif will be shown by transacions stream
         } catch (e) {
             console.log("Error happened while mortgaging stocks! ", e.statusCode, e.statusMessage, e);
```
Comment (gauthamk97) [inline public/src/components/mortgage/Mortgage.tsx]: Is this console log still supposed to be here?

File: public/src/components/mortgage/Mortgage.tsx [hunk 3]
```diff
@@ -137,7 +136,6 @@ export class Mortgage extends React.Component<MortgageProps, MortgageState> {
             retrieveStocksRequest.setStockQuantity(stockQuantity);
 
             const resp = await DalalActionService.retrieveMortgageStocks(retrieveStocksRequest, this.props.sessionMd);
-            console.log(resp.getStatusCode(), resp.toObject());
             showNotif("Stocks retrieved successfully");
         } catch (e) {
             console.log("Error happened while retrieving stocks! ", e.statusCode, e.statusMessage, e);
```
Comment (gauthamk97) [inline public/src/components/mortgage/Mortgage.tsx]: What about this one as well?

**Pull request merged**